tools/make_dictionary_file.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python2.7
+"""
+Makes dictionary file from mozc
+(dictionary*.txt in https://github.com/google/mozc/tree/master/src/data/dictionary_oss/)
+
+command line example: tools/make_dictionary_file.py dictionary*.txt
+
+The path to the output file: app/japanese_name_location_dict.txt
+"""
+import sys
+
+def check_number(id1, id2, numbers):
```

Comment:
Prefer to write "id1 in numbers and id1 == id2" directly in make_dictionary instead of defining this function.